.github/workflows/ci.yml

```
@@ -57,7 +57,10 @@ jobs:
           fetch-depth: 0
       - name: Install avocado libs
         working-directory: ./avocado-libs
-        run: pip install -e .
+        run: |
+         pip install --upgrade pip
```

As suggested by the [pip doc](https://pip.pypa.io/en/stable/installation/#upgrading-pip):
```suggestion
         python -m pip install --upgrade pip
```

---

@luckyh 
updated as suggested